core/src/main/java/com/scalar/db/api/PutBuilder.java

```diff
@@ -220,6 +223,24 @@ public Buildable implicitPreReadEnabled(boolean implicitPreReadEnabled) {
       return this;
     }
 
+    @Override
+    public Buildable disableInsertMode() {
+      insertModeEnabled = false;
+      return this;
+    }
+
+    @Override
+    public Buildable enableInsertMode() {
+      insertModeEnabled = true;
+      return this;
+    }
+
+    @Override
+    public Buildable insertModeEnabled(boolean insertModeEnabled) {
+      this.implicitPreReadEnabled = insertModeEnabled;
```

Comment:
```suggestion
      this.insertModeEnabled = insertModeEnabled;
```
Typo?

Reply:
Ah good catch! Thanks. Fixed in b745e97cd911b17c0802512a2024eead068c5cc6.